multiproof.go

```diff
@@ -24,6 +24,12 @@ type MultiProof struct {
 func CreateMultiProof(transcript *common.Transcript, ipaConf *ipa.IPAConfig, Cs []*banderwagon.Element, fs [][]fr.Element, zs []uint8) (*MultiProof, error) {
 	transcript.DomainSep("multiproof")
 
+	for _, f := range fs {
+		if len(f) != common.VectorLength {
+			return nil, fmt.Errorf("polynomial length = %d, while expected length = %d", len(f), common.VectorLength)
+		}
+	}
```

Comment:
Other bugs found by the fuzzer: if we don't check this, then L74 (i.e: `fs[i][j]`) can panic.